management/server/user.go

```diff
@@ -207,10 +215,14 @@ func (am *DefaultAccountManager) createServiceUser(accountID string, initiatorUs
 	if executingUser == nil {
 		return nil, status.Errorf(status.NotFound, "user not found")
 	}
-	if executingUser.Role != UserRoleAdmin {
+	if !executingUser.IsAdmin() {
 		return nil, status.Errorf(status.PermissionDenied, "only admins can create service users")
```

Comment:
I don't know if we consider the owner as a subset of the admin role. But maybe the error log needs to be adjusted,